proto/osmosis/incentives/gauge.proto

```diff
@@ -41,4 +41,45 @@ message LockableDurationsInfo {
     (gogoproto.stdduration) = true,
     (gogoproto.moretags) = "yaml:\"lockable_durations\""
   ];
+}
+
+message HistoricalReward {
```

Comment:
Lets add some comments to each of these proto structs.

This one seems clear -- Its the historical accumulator values, for every time someone starts something that is earning rewards.

Reply:
Comments added!